src/main/java/Event.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Represents an event to be recorded. An <code>Event</code> object
+ * is represented by three Strings
+ * e.g., <code>project meeting, Mon 2pm, to 4pm</code>
+ */
+public class Event extends Task {
+
+    protected String startTime;
+    protected String endTime;
+
+
+    public Event(String description, String startTime, String endTime) {
+        super(description);
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+
+    public Event(String description, String startTime, String endTime,
+                 boolean isDone) {
+        super(description, isDone);
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+
+    /**
+     * Returns summary of task
+     *
+     * @return task summary
+     */
+    @Override
+    public String summary() {
```

Comment:
nit: for method names try to use verbs instead (from textbook), maybe something like getSummary() will be better